examples/httpServer.go

```diff
@@ -10,7 +10,8 @@ import (
 	"os"
 
 	//"github.com/pkg/profile"
-	"gopkg.in/jcmturner/gokrb5.v6/credentials"
+	// "gopkg.in/jcmturner/gokrb5.v6/credentials"
```

Comment:
Can you remove this line rather than comment out. Just makes for cleaner code. Thanks.

Reply:
Sure, +1 on that.